File: lab07/count_word.py
#!/usr/bin/python
import re,sys,glob
count=0
reg= r"\b"+re.escape(sys.argv[1])+r"\b"
for line in sys.stdin:
	line = line.lower()
	array = re.findall(reg,line)
	count += len(array)
print sys.argv[1] + " occurred " + str(count) + " times"

File: lab07/count_word.pl
#!/usr/bin/perl

my $total=0;
foreach $line (<STDIN>) {
	$line = lc $line;
        @array = $line =~ /\b$ARGV[0]\b/g;   #match a word boundary
        $total += $#array +1;
}
print "$ARGV[0] occurred $total times\n";

